docs/howtos/telemetry/30-metrics-qs.md

```diff
@@ -98,8 +98,8 @@ EOF
 We can now install Kubewarden in the recommended way with Helm charts.
 
 :::note
-cert-manager is a requirement of Kubewarden, and OpenTelemetry is required for this
-feature, but we've already installed them in a previous section of this book.
+cert-manager is a requirement of OpenTelemetry, but we've already installed
+them in a previous section of this book.
```

Comment:
Yes, we have. But not because of Kubewarden. OpenTelemetry uses CertManager to generate certificates by default. https://deploy-preview-446--silly-bunny-8cedd0.netlify.app/howtos/telemetry/opentelemetry-qs#install-opentelemetry